config/checkstyle_checks.xml

```diff
@@ -81,40 +81,50 @@
   <module name="OrderedProperties" />
 
   <!-- Regexp -->
-  <module name="RegexpMultiline"/>
   <module name="RegexpMultiline">
+    <property name="id" value="regexpMultilineDefault"/>
+  </module>
+  <module name="RegexpMultiline">
+    <property name="id" value="noConsecutiveLines"/>
     <property name="format" value="\r?\n[\t ]*\r?\n[\t ]*\r?\n"/>
     <property name="fileExtensions" value="java,xml,properties"/>
     <property name="message" value="Unnecessary consecutive lines"/>
   </module>
   <module name="RegexpMultiline">
+    <property name="id" value="commentFirstSentenceMultiline"/>
     <property name="format" value="/\*\*\W+\* +\p{javaLowerCase}"/>
     <property name="fileExtensions" value="java"/>
     <property name="message"
               value="First sentence in a comment should start with a capital letter"/>
   </module>
   <module name="RegexpMultiline">
+    <property name="id" value="noEmptyFile"/>
     <property name="format" value="^\s*$" />
     <property name="matchAcrossLines" value="true" />
     <property name="message" value="Empty file is not allowed" />
   </module>
   <module name="RegexpSingleline">
+    <property name="id" value="noEmptyLineWithWhitespace"/>
```

Comment:
This is not  about empty lines. Any string with a trailing space is not allowed. I.e.
```java
if (someCondition) {   
```
three trailing spaces here ^^^

Please change to `noTrailingWhitespace` and "Trailing whitespace is not allowed"

Reply:
done @pbludov